tests/ha/single_pod_restart/test_data_pod_restart.py

```diff
@@ -97,26 +96,23 @@ def setup_class(cls):
 
         cls.rest_obj = S3AccountOperations()
         cls.s3_mp_test_obj = S3MultipartTestLib(endpoint_url=S3_CFG["s3_url"])
-        cls.test_file = "ha-mp_obj"
         cls.test_dir_path = os.path.join(TEST_DATA_FOLDER, "HATestMultipartUpload")
```

Comment:
do we need this? please check i guess this is require in api files

Reply:
test_file is not used in current tests
test_dir_path is used at many places